CHANGELOG.md

```diff
@@ -55,6 +55,8 @@
 * [ENHANCEMENT] Experimental Delete Series: Retry processing of Delete requests during failures. #2926
 * [ENHANCEMENT] Improve performance of QueryStream() in ingesters. #3177
 * [ENHANCEMENT] Modules included in "All" target are now visible in output of `-modules` CLI flag. #3155
+* [ENHANCEMENT] Add flag `-<prefix>.tls-server-name` to require a specific server name instead of the hostname on the certificate. #3156
+* [ENHANCEMENT] Allow configuration of non-mutual TLS by using the flag `-<prefix>.tls-enabled`. #3156
```

Comment:
I've just merged a PR to cut the CHANGELOG because of the upcoming `1.4.0` release. Could you rebase `master` and move your CHANGELOG entry to the top, under `## master / unreleased`, please?